### DISCUSSION TITLE
How do I make so that Parcel always build the output as an ES module?

### DISCUSSION THREAD

**Discussion by lines-of-codes** [parcel-bundler/parcel#7624] [Q&A]
So, I had this little problem here.
I wanted Parcel to build my TypeScript ES module file as a JavaScript ES module file and not a JavaScript CommonJS module file. Which CLI arguments do I need to use?

Also, I am compiling multiple files.

**Comment by devongovett** [comment]
Take a look at the docs for targets. In particular, the outputFormat option: https://parceljs.org/features/targets/#outputformat
